src/components/Contentful/HeroWithCarousel.vue

```diff
@@ -31,6 +31,12 @@
 							</template>
 						</kv-carousel>
 					</div>
+					<div class="tw-col-span-12 tw-text-center">
```

Comment:
Should this be conditional so it doesn't introduce the extra margin or any other space from the empty containers?

Reply:
Or maybe the ButtonWrapper just won't render it's container due it's internal `v-if`...that would be ok too!

Reply:
I just tested this it actually renders as 0 height, so it won't add any space, and the button wrapper won't render if there is no button content, so no v-if required